nuxeo-features/nuxeo-automation/nuxeo-automation-core/src/main/java/org/nuxeo/ecm/automation/core/work/BlobListZipWork.java

```diff
@@ -43,7 +43,9 @@
  * Work to zip a list of document default blob and store the produced zip into the TransientStore.
  *
  * @since 9.3
+ * @deprecated since 10.3
```

Comment:
please add a comment, e.g : not used anymore since download service is synchronous